crowbar_framework/lib/remote_node.rb

```diff
@@ -97,7 +97,7 @@ def pipe_ssh_cmd(host_or_ip, cmd, stdin)
   # chef-client daemon started by pid 1 is not counted
   def chef_clients_running?(host_or_ip)
     ssh_cmd = ssh_cmd_base(host_or_ip)
-    ssh_cmd << "'if test -f /var/chef/cache/chef-client-running.pid; then flock -n /var/chef/cache/chef-client-running.pid true; fi'"
+    ssh_cmd << "'if test -f /var/chef/cache/chef-client-running.pid; then flock -n /var/chef/cache/chef-client-running.pid true; elif test -f /var/cache/chef/chef-client-running.pid; then flock -n /var/cache/chef/chef-client-running.pid true; fi'"
```

Comment:
Metrics/LineLength: Line is too long. [247/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)